R/mainTests.r

```diff
@@ -1292,27 +1366,13 @@ school_laestab_duplicates <- function(data) {
 
 # school_urn_duplicates  ----------------------------------------
 # check that there is a 1:1 relationship between school urns and names
+
```

Comment:
same as above for legitimate duplicate code sharing

Reply:
Don't think there should be any duplicate URN's, think they will always be an error?

Reply:
Resolving this as now removing these checks entirely (the platform should be fine with duplicates like these)